tests/test_plotting.py

```diff
@@ -128,5 +184,15 @@ def test_legend(self):
         self.assertEqual(labels, expected)
 
 
+def _check_colors(collection, expected_colors):
+
+    import matplotlib.colors as colors
+    conv = colors.colorConverter
+
+    for patch, color in zip(collection, expected_colors):
+        result = patch.get_color()
+        assert conv.to_rgba(result) == conv.to_rgba(color)
```

Comment:
Aha, but it is not from this PR, but after merging #228. We actually should have rerun the test to catch that. 
The `colormap` arguments should also be changed to `cmap` in the tests, which is a simple change. Will do a PR